package.json

```diff
@@ -25,7 +25,7 @@
     "lint": "eslint src --ext .ts --fix",
     "prepublishOnly": "run-s build",
     "preversion": "run-s lint format:check test build",
-    "test": "vitest run",
```

Comment:
I'll have to revert this, as preversion script requires a test run to pass or fail